modules/nw-sriov-supported-devices.adoc

```diff
@@ -81,4 +86,9 @@
 |MT28908 Family [ConnectX&#8209;6{nbsp}Lx]
 |15b3
 |101f
-|===
\ No newline at end of file
+|===
+
+[NOTE]
+====
+For the most up to date list of supported cards and compatible {product-title} versions available, refer to link:https://access.redhat.com/articles/6954499[Openshift Single Root I/O Virtualization (SR-IOV) and PTP hardware networks Support Matrix].
```

Comment:
`refer to -> see`
IBM style guide recommends using 'see' when referencing within a document or other material